js/state/syncUtil.js

```diff
@@ -472,13 +523,13 @@ const findOrCreateFolderObjectId = (folderId, appState) => {
     const AppStore = require('../stores/appStore')
     appState = AppStore.getState()
   }
-  const folder = appState.getIn(['sites', folderId.toString()])
+  const folder = appState.getIn(['bookmarkFolders', folderId.toString()])
   if (!folder) { return undefined }
   const objectId = folder.get('objectId')
   if (objectId) {
     return objectId.toJS()
   } else {
-    return module.exports.newObjectId(['sites', folderId.toString()])
+    return module.exports.newObjectId(['bookmarkFolders', folderId.toString()])
```

Comment:
Can we create constants file like `siteTags`. Because this way we are less error prone.